pommerman/utility.py

```diff
@@ -328,3 +331,43 @@ def get_next_position(position, direction):
 
 def make_np_float(feature):
     return np.array(feature).astype(np.float32)
+
+def join_json_state(record_json_dir, agents, finishedAt, config):
+    jsonSchema = {
+        "properties": {
+            "state": {
+                "mergeStrategy": "append"
+            }
+        }
+    }
+
+    jsonTemplate = {
+        "agents": agents,
+        "finishedAt": finishedAt,
+        "config": config,
+        "state": []
+    }
+
+    merger = Merger(jsonSchema)
+    base = merger.merge({}, jsonTemplate)
+
+    for root, dirs, files in os.walk(record_json_dir):
+        for name in files:
+            path = os.path.join(record_json_dir, name)
+            if name.endswith('.json') and "game_state" not in name:
+                with open(path) as data_file:    
+                    data = json.load(data_file)
+                    head = {"state":[data]}
+                    base = merger.merge(base, head)
+    
+    filename = 'game_state_' + datetime.now().strftime('%Y%m%d%H%M%S') + '.json'
```

Comment:
what's the different between this datetime and finished_at?

Reply:
Nothing. TBH if each episode is in it's own sub dir then we can remove it.